docs/user_guide/vis/index.rst

```diff
@@ -1,88 +1,264 @@
+
+The *diagram* (or *flowsheet*) area lets you rearrange the flowsheet as you need and zoom in on particular sections.
+You can interact with the components on the diagram:
+
+Shapes
+    Geometric shapes on the flowsheet represent unit models, inlets and outlets, and other IDAES components.
+    They are connected by lines, and each has a name. All shapes can be moved by clicking and dragging them.
+    If you right-click on a shape, it will rotate 90 degrees. You cannot add or remove lines, as these
+    are determined by the underlying flowsheet.
+
+Lines
+    The lines connecting units can be manipulated by clicking and dragging.
+    You can click on a line to create a new segment that can be used for routing the line
+    around objects. You can eliminate a segment by clicking on the dot that appears as you hover over
+    the line. There are also pill-shaped handles that appear on the lines for moving them.
+    The endpoints of the lines are determined by the flowsheet and cannot be changed.
+
+Labels
+    Both the units and the arcs have associated values that can be shown, which pop up over the
+    lines if you toggle the "Show labels" control. See the :ref:`diagram-controls` section for details.
+
+:ref:`Back to main window screenshot <ifv-screenshot>`
+
+.. _diagram-controls:
+
+Diagram Controls
+++++++++++++++++
+
+.. figure:: ../../_images/ifv_screenshot_diagramcontrols.png
+    :width: 400
+
+    Screenshot of the *diagram controls* area of the IFV UI
+
+The *diagram controls* allow you to affect some global properties of the diagram/flowsheet area.
+
+.. |zoomin| image:: ../../_images/ifv_icon_zoomin.png
+.. |zoomout| image:: ../../_images/ifv_icon_zoomout.png
+.. |zoomfit| image:: ../../_images/ifv_icon_fit.png
+
+View actions
+  * Labels: Toggle view of the information (*labels*) shown for each stream. This is the same information
+    that appears in the :ref:`ifv_stream-table`.
+  * Grid: Toggle a background "grid"
+  * |zoomin|: Zoom in by 25%
+  * |zoomout|: Zoom out by 25%
+  * |zoomfit|: Fit the diagram into the current area
+
+:ref:`Back to main window screenshot <ifv-screenshot>`
+
+.. _ifv_streamtable:
+
+Stream Table
+++++++++++++
+
+The IFV will show a stream table with variables defined for each stream in the flowsheet, if these
+values exist and the flowsheet adheres to the IDAES conventions for naming the inlet and outlet
+streams. An example of a stream table is shown below.
+
+.. figure:: ../../_images/ifv_screenshot_streamtable.png
+    :width: 800
+
+    Screenshot of an example stream table
+
+There are a number of ways of manipulating this table:
+
+  * The "Hide Fields" pull-down menu provides a list of stream names.
+    Select a name to hide/show that column in the table.
+  * Click on the column header and drag it left or right to change its order in the table.
+  * Resize a column by hovering over a column border until you see the mouse pointer change, then drag it to resize.
```

Comment:
You can also double click to have the column auto-resize

Reply:
You have to double click on when the mouse pointer changes not just double click on the column. Sorry I should have specified that.